### PR TITLE
Cy

### PR DESCRIPTION
**Pull request by moringspeaker**
Add business displaying pages and change the global max-width to adjast some special screens like mine

### CODE REVIEW

File: users/urls.py
Comment (ws2309nyu) [inline users/urls.py:1]: this change is already made on develop

File: users/views.py
Comment (ws2309nyu) [inline users/views.py:1]: this change is already made on develop

File: static/CACHE/css/users.860e57148f43.css
Comment (ws2309nyu) [inline static/CACHE/css/users.860e57148f43.css:1]: Is uploading a css cache really needed?

File: static/CACHE/css/base.c17079664a40.css
Comment (ws2309nyu) [inline static/CACHE/css/base.c17079664a40.css:1]: Is uploading a css cache really needed?

File: static/CACHE/css/bootstrap.f02b45cd7853.css
Comment (ws2309nyu) [inline static/CACHE/css/bootstrap.f02b45cd7853.css:1]: Is uploading a css cache really needed?

File: static/CACHE/css/homepage.c233a0339f3e.css
Comment (ws2309nyu) [inline static/CACHE/css/homepage.c233a0339f3e.css:1]: Is uploading a css cache really needed?

File: static/CACHE/css/users.5507f58781c3.css
Comment (ws2309nyu) [inline static/CACHE/css/users.5507f58781c3.css:1]: Is uploading a css cache really needed?

**Review by ws2309nyu — COMMENTED** [review]
Added some comments on CSS Cache uploads

**Review by ws2309nyu — APPROVED** [review]
Approved url test and views test. Maybe Stephanie can check and approve the front end additions.

**Comment by vchrombie** [comment]
Most of the work here is already merged on develop. I guess we can close this PR.

Please feel free to open if you have any other thoughts.

**Pull request closed without merging**